website/content/_index.md

```diff
@@ -16,22 +16,86 @@ Wash helps you deal with all your remote or cloud-native infrastructure using th
+
+<div style="width:200px">List all</div> | 
+----------------------|---
+AWS EC2 instances     | `find aws/foo -k '*ec2*instance'`
+Docker containers     | `find docker -k '*container' `
+Kubernetes pods       | `find kubernetes -k '*pod'`
+GCP Compute instances | `find gcp -k '*compute*instance'`
+
+<div style="width:200px">Read</div>         | 
+--------------------------------------------|---
+Console output of an EC2 instance           | `cat aws/foo/resources/ec2/instances/bar/console.out`
+Console output of a Google compute instance | `cat gcp/<project>/compute/foo/console.out`
+An S3 object's content                      | `cat aws/foo/resources/s3/bar/baz`
+A GCP Storage object's content              | `cat gcp/<project>/storage/foo/bar`
+
+<div style="width:200px">Exec `uname` on </div> | 
+-----------------------------|---
+An EC2 instance              | `wexec aws/foo/resources/ec2/instances/bar uname`
+An a Docker container        | `wexec docker/containers/foo uname`
+Exec on a Kubernetes pod     | `wexec kubernetes/<context>/<namespace>/pods/foo uname`
+On a Google Compute instance | `wexec gcp/<project>/compute/foo uname`
+
+<div style="width:200px">Find by 'owner' tag/label</div> | 
+------------------------|---
+EC2 instances           | `find aws/foo -k '*ec2*instance' -meta '.tags[?].key' owner`
+Docker containers       | `find docker -k '*container' -meta '.labels.owner' -exists`
+Kubernetes pods         | `find kubernetes -k '*pod' -meta '.metadata.labels.owner' -exists`
+Google Compute instance | `find gcp -k '*compute*instance' -meta '.labels.owner' -exists`
+
+From the table, we see that using Wash means:
+
+* You no longer have to learn different commands to execute a task across different things. All you need is one command (`find` for List/Find; `cat` for Read; and `wexec` for Exec).
+
+* You no longer have to install a bunch of different tools. All you need to install is the Wash binary.
+
+* You no longer have to learn different DSLs for filtering stuff. All you need to learn is find's expression syntax and its individual primaries. Once you do that, you can filter on almost any conceivable property of your specific thing.
+
+And this is only scratching the surface of Wash's capabilities. Checkout the screencast below to see some more (and to see Wash in action):
 
-Wash aims to:
-
-* help you keep track of what systems you're running
-* make scripting across your new-fangled infrastructure as easy as writing a local shell script
-* provide easy access to files, logs, metadata, and any other activity for viewing, editing, and UNIXy slicing-and-dicing
-* help you build better tools that work across a variety of cloud APIs
-* let you easily build your own integrations in whatever language you want
-* be extremely simple to get up-and-running; if it takes you more than a few minutes, let us know!
+<script id="asciicast-mX8Mwa75rr1bJePLi3OnIOkJK" src="https://asciinema.org/a/mX8Mwa75rr1bJePLi3OnIOkJK.js" async></script>
```

Comment:
Going to spend tomorrow thinking about some ideas on how to improve the screencast. Main thing is to show some of the listed invocations of Table 2 in action.

I think with this intro, we can get away with a slightly longer screencast (to showcase some of the cooler features of Wash, and to also spend time explaining some of its concepts).